acceptance/tests/options/config_file/custom_facts.rb

```diff
@@ -1,61 +1,42 @@
-# This test is intended to demonstrate that the directory for custom facts can be specified from
-# the config file using the global.custom-dir field, and that custom fact collection can be disabled
-# by setting the global.no-custom-facts field to true.
-test_name "custom-dir and no-custom-facts config fields allow control of custom fact lookup" do
+# This test verifies that facter can load facts from a single custom-dir specified
+# in the configuration file
+test_name "C98143: config custom-dir allows single custom fact directory" do
+  tag 'risk:low'
+
   require 'facter/acceptance/user_fact_utils'
   extend Facter::Acceptance::UserFactUtils
 
-  custom_fact_content = <<EOM
-Facter.add('custom_fact') do
+  content = <<EOM
+Facter.add('config_fact') do
   setcode do
-   "testvalue"
+    "config_value"
   end
 end
 EOM
 
   agents.each do |agent|
-    custom_dir = get_user_fact_dir(agent['platform'], on(agent, facter('kernelmajversion')).stdout.chomp.to_f)
-
-    step "Agent #{agent}: create custom fact and config file" do
-      on(agent, "mkdir -p '#{custom_dir}'")
+    step "Agent #{agent}: create custom fact directory and a custom fact" do
+      custom_dir = agent.tmpdir('custom_dir_1')
```

Comment:
Because it was a cut and paste of the multiple directory version and I missed changing the name. I'll fix it